draft-ietf-quic-http.md

```diff
@@ -1039,7 +1044,8 @@ request that was sent, whether in whole or in part, might have been processed.
 
 A QUIC stream provides reliable in-order delivery of bytes, but makes no
 guarantees about order of delivery with regard to bytes on other streams. On the
-wire, data is framed into QUIC STREAM frames, but this framing is invisible to
+wire, data (HTTP frames)
+is framed into QUIC STREAM frames, but this framing is invisible to
```

Comment:
I think that use of "data" is confusing, but I might prefer:

```suggestion
wire, HTTP frames are
framed into QUIC STREAM frames, but this framing is invisible to
```

Reply:
That looks better than what I have, but "foo frames are framed into" is also a bit awkward, so I'll leave it open for a bit.

Reply:
How about...
```suggestion
wire, the stream data containing HTTP frames
is carried by QUIC STREAM frames, but this framing is invisible to
```
I want to be careful to avoid the implication that HTTP frames need to be contained within a single STREAM frame.